packages/shopping/src/__tests__/e2e/integration/cart.spec.js

```diff
@@ -0,0 +1,80 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback4-example-shopping
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+context('Cart', () => {
+  before(() => cy.visit('http://localhost:3000/shoppy.html'));
+
+  it('should add items to Cart', () => {
+    addToCart();
+  });
+
+  it('should checkout all items from Cart', () => {
+    cy.get('[data-cy=shoppingCartLink]')
+      .should('be.visible')
+      .click()
+      .then(() => {
+        cy.get('[data-cy=logInButton]').click();
+
+        cy.get('[data-cy=shoppingCartLink]').should('be.visible').click();
```

Comment:
I can't reply to the old thread for some reason, but this test is still failing for me. The login modal is still open, so despite the link being visible, it can't be clicked on:

![image](https://user-images.githubusercontent.com/42985749/96880963-c4b17900-144b-11eb-9b35-e5fb6ae7497a.png)

I think you'll also have to verify that the modal not visible.

Reply:
Difficult to fix this 1 on my end since the test suite is passing. How are you executing these?

Reply:
I'm using the "Run all specs" option

Reply:
<img width="347" alt="Screenshot 2020-10-22 at 17 01 45" src="https://user-images.githubusercontent.com/5213555/96890860-5dfe8080-1488-11eb-8672-3aad63f099e9.png">

Reply:
<img width="349" alt="Screenshot 2020-10-22 at 17 07 29" src="https://user-images.githubusercontent.com/5213555/96891578-17f5ec80-1489-11eb-9539-22b74df48fc5.png">